.changeset/fast-moments-mix.md

```diff
@@ -3,10 +3,10 @@
 "@fluidframework/tree": minor
 "__section": tree
 ---
-Fix typing derived from unions of `AllowedTypes` arrays
+Typing derived from unions of AllowedTypes arrays are fixed
 
 Unions of array types provided as an [AllowedTypes](https://fluidframework.com/docs/api/fluid-framework/allowedtypes-typealias) used to result in incorrectly computed insertable content types.
-This happened because [InsertableTreeNodeFromAllowedTypes](https://fluidframework.com/docs/api/fluid-framework/insertabletreenodefromallowedtypes-typealias) distributed over the union, violating the policy documented in [Input](https://fluidframework.com/docs/api/fluid-framework/input-typealias) for how schema derived input types should be computed.
+This happened because [InsertableTreeNodeFromAllowedTypes](https://fluidframework.com/docs/api/fluid-framework/insertabletreenodefromallowedtypes-typealias) distributed over the union, violating the policy documented in [Input](https://fluidframework.com/docs/api/fluid-framework/input-typealias) for how schema-derived input types should be computed.
 This has been fixed.
-To get usable Input types, SharedTree schema's types should always capture the exact schema provided at runtime and not unions of possible different schema.
+To get usable Input types, SharedTree schemas' types should always capture the exact schema provided at runtime and not unions of possible different schema.
```

Comment:
I _think_ this is the correct plural possessive. "schema" can be singular or plural, so maybe the existing punctuation was correct, but if nothing else I think this signals more explicitly that we're using the plural form? Happy to revert if someone knows better.